types/index.d.ts

```
@@ -1046,142 +1003,180 @@ export declare function assert(received: any): Assertion;
+/**
+ * Creates a tracked version of a given function.
+ *
+ * @param {Function} implementation - The original function implementation.
+ * @returns {TrackFn & Function} A tracked version of the provided function.
+ *
+ * @example
+ * const add = (a: number, b: number) => a + b;
+ * const trackedAdd = Fn(add);
+ * trackedAdd(1, 2); // 3
+ */
+export declare function Fn(implementation: Function): Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)